### ISSUE TITLE
Doesn't work on firefox, throws error "ReferenceError: IntersectionObserver is not defined". Doesn't work with dom-repeat on chrome.

### ISSUE THREAD

**Issue by rraagghhu** [1000ch/lazyload-image#11]

**Comment by myfreeweb** [comment]
Yes, Firefox doesn't implement IntersectionObserver. There's [a polyfill](https://github.com/WICG/IntersectionObserver/tree/gh-pages/polyfill) but… it's not very useful. Using the web components polyfill, the `lazyload-image` element gets loaded too late to prevent initial loading of the images.
